reactorch/flame_1d.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Weiqi Ji"
+__copyright__ = "Copyright 2020, DENG"
+
+__version__ = "0.1"
+__email__ = "weiqiji@mit.edu"
+__status__ = "Development"
+
+import torch
+
+
+torch.set_default_tensor_type("torch.DoubleTensor")
+
+
+class Flame_1d:
+    def __init__(self, tct, device=None):
+        super().__init__()
+        
+        if device is None:
+            self.device = torch.device('cpu')
+        else:
+            self.device = device
+        
+        self.tct = tct
```

Comment:
This is also not urgent.
We can change .tct to .sol in this file, to better connect the name with the class Solution.